src/components/Nav.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react'
 import Link from 'next/link'
-import { useRouter } from 'next/router'
+import Router, { useRouter } from 'next/router'
```

Comment:
You can use existing `useRouter()` which expose the same API with `Router` 
Ref: https://nextjs.org/docs/api-reference/next/router#router-object